sdk/src/main/java/com/meniga/sdk/helpers/ResultMapper.kt

```diff
@@ -0,0 +1,18 @@
+package com.meniga.sdk.helpers
+
+import com.meniga.sdk.providers.tasks.Task
+
+internal class ResultMapper<T, R>(
```

Comment:
Could you add a short class description to this class. It's not clear to me how it's used just by looking at it :)

Reply:
Will do that :).